src/material/chips/chip.ts

```diff
@@ -210,6 +217,9 @@ export class MatChip
   /** The unstyled chip selector for this component. */
   protected basicChipAttrName = 'mat-basic-chip';
 
+  /** Default chip options. */
+  protected _defaultOptions = inject(MAT_CHIPS_DEFAULT_OPTIONS, {optional: true});
```

Comment:
This is only being used in the chip option so it should be moved there.

Reply:
Done I moved all the `_defaultOptions` in this PR so that they're only accessible in the classes that use them.